cognite/client/_api_client.py

```diff
@@ -54,34 +55,16 @@ class APIClient:
     _RESOURCE_PATH: str
 
     # TODO: This following set should be generated from the openapi spec somehow.
-    RETRYABLE_POST_ENDPOINTS = {
-        "/assets/list",
-        "/assets/byids",
-        "/assets/search",
-        "/events/list",
-        "/events/byids",
-        "/events/search",
-        "/files/list",
-        "/files/byids",
-        "/files/search",
+    _RETRYABLE_POST_ENDPOINT_REGEX_PATTERNS = {
+        "/(assets|events|files|timeseries|sequences|datasets|relationships)/(list|byids|search|aggregate)",
         "/files/downloadlink",
-        "/timeseries/byids",
-        "/timeseries/search",
-        "/timeseries/list",
         "/timeseries/data",
         "/timeseries/data/list",
         "/timeseries/data/latest",
         "/timeseries/data/delete",
-        "/sequences/byids",
-        "/sequences/search",
-        "/sequences/data",
```

Comment:
this one disappeared

Reply:
They're included in the first regex pattern.

Reply:
thanks